### ISSUE TITLE
DAISY Pipeline missing common Alt key combinations for file, edit, and help

### ISSUE THREAD

**Issue by GeorgeKerscher** [daisy/pipeline-ui#59]
Most Windows application have shortcut keys for the major categories. Currently, these are missing in the DAISY Pipeline. You can get there by hitting Alt and then arrowing around, but it would be good to use traditional conventions in the interface.

**Comment by NPavie** [comment]
fixed by 08dccdb
